google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/ConflictingAppEnginePluginCheck.java

```diff
@@ -109,6 +115,9 @@ public void hyperlinkUpdate(@NotNull Notification notification, @NotNull Hyperli
       String href = event.getDescription();
 
       if (DEACTIVATE_LINK_HREF.equals(href)) {
+        UsageTrackerProvider.getInstance()
+            .trackEvent(GctTracking.APP_ENGINE_OLD_PLUGIN_NOTIFICATION_CLICK)
```

Comment:
Oh is the notification_click clicking on the warning to open up the "are you sure you want to disable.." dialog?